aviary/docs/user_guide/SGM_capabilities.ipynb

```diff
@@ -0,0 +1,220 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aviary.mission.gasp_based.phases.time_integration_traj import FlexibleTraj\n",
+    "from aviary.interface.default_phase_info.two_dof_fiti import phase_info, add_default_sgm_args\n",
+    "\n",
+    "add_default_sgm_args(phase_info, ode_args)\n",
+    "\n",
+    "full_traj = FlexibleTraj(\n",
+    "    Phases=phase_info,\n",
+    "    traj_final_state_output=[\n",
+    "        Dynamic.Mission.MASS,\n",
+    "        Dynamic.Mission.DISTANCE,\n",
+    "    ],\n",
+    "    traj_initial_state_input=[\n",
+    "        Dynamic.Mission.MASS,\n",
+    "        Dynamic.Mission.DISTANCE,\n",
+    "        Dynamic.Mission.ALTITUDE,\n",
+    "    ],\n",
+    "    traj_event_trigger_input=[\n",
+    "        # specify ODE, output_name, with units that SimuPyProblem expects\n",
+    "        # assume event function is of form ODE.output_name - value\n",
+    "        # third key is event_idx associated with input\n",
+    "        ('groundroll', Dynamic.Mission.VELOCITY, 0,),\n",
+    "        ('climb3', Dynamic.Mission.ALTITUDE, 0,),\n",
+    "        ('cruise', Dynamic.Mission.MASS, 0,),\n",
+    "    ],\n",
+    "    traj_intermediate_state_output=[\n",
+    "        ('cruise', Dynamic.Mission.DISTANCE),\n",
+    "        ('cruise', Dynamic.Mission.MASS),\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because all phases require `ode_args` and `simupy_args` which are usually the same for all phases, `add_default_sgm_args` has been provided to add these to the phase info automatically.\n",
+    "\n",
+    "### Setting up Phase Info\n",
+    "\n",
+    "SGM uses the `2dof_fiti` (two degree of freedom, forward in time integration) phase_info, which contains the information required to build the default trajectory used by GASP. This phase info can be imported all at once or in a few pre-defined groups:\n",
+    "phase_info contains all the phases from ascent_phases (which is composed of takeoff_phases and climb_phases), cruise_phases, and descent_phases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aviary.interface.default_phase_info.two_dof_fiti import phase_info\n",
+    "from aviary.interface.default_phase_info.two_dof_fiti import takeoff_phases, climb_phases, descent_phases"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "phase_info_parameterization can be used to update the values of certain variables, like speed_trigger or cruise_alt using values from the input deck.\n",
+    "\n",
+    "## Idle Descent Estimation\n",
```

Comment:
Doesn't have to actually be idle though, right? Is that throttle hardcoded in?
Suggested change to "Descent Fuel Estimation" if idle isn't required